test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -96,6 +104,169 @@ public void CanChangeEngineFromScriptEditorDropDown()
             DispatcherUtil.DoEvents();
         }
 
+        /// <summary>
+        /// This test will validate the OnTextAreaTextEntering event that is generated by the ScriptEdirtorWindows when doing auto-completion
+        /// </summary>
+        [Test]
+        public void OnTextAreaTextEnteringEventTest()
+        {
+            // Arrange
+            string pythonCode = "from System.Collections import ArrayList";
+            Open(@"core\python\python.dyn");
+
+            var nodeView = NodeViewWithGuid("3bcad14e-d086-4278-9e08-ed2759ef92f3");
+            var nodeModel = nodeView.ViewModel.NodeModel as PythonNodeBase;
+            Assert.NotNull(nodeModel);
+
+            var scriptWindow = EditPythonCode(nodeView, View);
+            var codeEditor = FindCodeEditor(scriptWindow);
+            scriptWindow.editText.TextArea.TextEntering += TextArea_TextEntering;
```

Comment:
unsubscribe?

Reply:
@QilongTang I added the code for unsubscribing the event in the next commit:

https://github.com/DynamoDS/Dynamo/pull/11137/commits/362fc84368f0c7b5537c8378491a724f7a002a67